data/stoa0023/stoa001/__cts__.xml

```diff
@@ -2,10 +2,12 @@
             <ti:title xml:lang="lat">Res Gestae</ti:title>
             <ti:translation urn="urn:cts:latinLit:stoa0023.stoa001.perseus-eng2" workUrn="urn:cts:latinLit:stoa0023.stoa001" xml:lang="eng">
               <ti:label xml:lang="eng">History</ti:label>
-              <ti:description xml:lang="eng">Ammianus Marcellinus. Vols. I-III. Rolfe, John C., translator. London: W. Heinemann, 1935-1939 (printing).</ti:description>
+              <ti:description xml:lang="eng">Ammianus Marcellinus. Ammianus Marcellinus, with an English translation, Vols. I-III. Rolfe, John C., translator. Cambridge, MA; London: Harvard University
+                Press; W. Heinemann, 1935-1940.</ti:description>
             </ti:translation>
           <ti:edition urn="urn:cts:latinLit:stoa0023.stoa001.perseus-lat2" workUrn="urn:cts:latinLit:stoa0023.stoa001">
```

Comment:
If the date is after 1922 and it's a Loeb, we specifically are adding (printing) or (print) to the dates. This the practice we adopted with Plutarch and others.